packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -122,15 +129,10 @@ function Thread( {
 		debouncedToggleBlockHighlight( thread.blockClientId, false );
 	};
 
-	const handleCommentSelect = ( { id, blockClientId } ) => {
+	const handleCommentSelect = () => {
 		setShowCommentBoard( false );
-		setSelectedThread( id );
-		if ( blockClientId && relatedBlockElement ) {
-			relatedBlockElement.scrollIntoView( {
-				behavior: 'instant',
-				block: 'center',
-			} );
-		}
+		setSelectedThread( thread.id );
+		selectBlock( thread.blockClientId, null );
```

Comment:
Let's document what the second arg `null` does; otherwise, it's easy to forget.

Reply:
Updated in d5631121369c2992d4bb9f7a3e398dae9fdb8a28